homeassistant/components/unifiprotect/__init__.py

```diff
@@ -60,11 +60,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         nvr_info = await protect.get_nvr()
     except NotAuthorized as err:
         raise ConfigEntryAuthFailed(err) from err
-    except (asyncio.TimeoutError, NvrError, ServerDisconnectedError) as notreadyerror:
-        raise ConfigEntryNotReady from notreadyerror
+    except (asyncio.TimeoutError, NvrError, ServerDisconnectedError) as err:
+        raise ConfigEntryNotReady from err
 
     if nvr_info.version < MIN_REQUIRED_PROTECT_V:
-        _LOGGER.error(
+        _LOGGER.debug(
```

Comment:
I think it is fine to keep this as `error`.  They need to know why it didn't setup

Reply:
It should be in the error in the form. That is why I was thinking of keeping it debug. But I was split between the two. Did not want to pump out more logs then were needed.

Reply:
They could remove the older HACS integration and be on an older version of protect, then wonder why it doesn't work.

Reply:
Oh, crap. I just realized I change the _wrong one_ to debug. Haha. The one in the _config flow_ was suppose to be the debug one. I can still make them both error though.

Reply:
The config flow one is fine as `debug` since they get UI feedback.

This one should be `error`